src/Attendance/AttendanceOtherForm/AttendanceOtherForm.js

```diff
@@ -88,7 +88,7 @@ export function AttendanceOtherForm({
       const caseNoteSuffix = absentSubReasonOption
         ? `${absentSubReasonOption.name}. ${values.comments}`
         : values.comments
-      const iepValues = { pay: values.pay, caseNote: `${absentReasonOption.name}: ${caseNoteSuffix}` }
+      const iepValues = { pay: values.pay, caseNote: `${absentReasonOption.name} - ${caseNoteSuffix}` }
```

Comment:
this is now consistent with whereabouts - https://github.com/ministryofjustice/whereabouts-api/blob/c83c0a291ac639663559126461f83b5f352f981e/src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/IEPWarningService.kt#L91